source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -166,13 +166,50 @@ Then import the JavaScript file before the closing `</body>` tag of your HTML pa
 
 ### Import JavaScript using a bundler
 
-If you decide to import using a bundler, use `import` to import all of GOV.UK Frontend's components, then run the `initAll` function to initialise them:
+If you decide to import using a bundler, we recommend you use `import` to only import the JavaScript for components you're using in your service. For example:
+
+```javascript
+import { SkipLink, Radios } from 'govuk-frontend'
+
+var $skipLink = document.querySelector('[data-module="govuk-skip-link"]')
+if ($skipLink) {
+  new SkipLink($skipLink).init()
+}
+
+var $radios = document.querySelectorAll('[data-module="govuk-radios]')
+if ($radios) {
+  for (var i = 0; i < $radios.length; i++) {
+    new Radios($radios[i]).init()
+  }
+}
+```
+
+If you need to import all of GOV.UK Frontend's components, then run the `initAll` function to initialise them:
 
 ```javascript
 import { initAll } from 'govuk-frontend'
 initAll()
 ```
 
+If you're using Webpack, you may need to add the following to your Webpack config file:
```

Comment:
Less sure about this section, as we might need to keep an eye on it and maintain it. But it seems pretty necessary for users using Webpack, so we might just have to take on that burden for a bit.

Reply:
Are we fairly confident that this is pretty much exactly what users will need to add, or should we soften it to something like "...some or all of the following..."?

Reply:
Yeah, good point, I wouldn't say I'm confident on anything Webpack related. I'm pretty sure this bit is needed (both you and Jani needed to add it when testing with Webpack), but there may be additional config not included here. I'll try to soften it a bit.